### DISCUSSION TITLE
Pocketbase as an OAuth 2 provider?

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1144] [Q&A] [answered]
Would it ever be possible for Pocketbase to have a built-in option to use it as a OAuth2 provider?  So for example PB could allows users via Google oauth2, raw signup with password and then other sites in the network could use the PB server as a central authority.

Am I crazy talking again?  Pocketbase already does this right?  ;)

**Comment by ganigeorgiev** [comment, marked as answer]
No this is not possible at the moment and at least for v1 there are no plans to allow using PocketBase as OAuth2 provider since it will complicate too much the API (I'm not sure how access controls and permissions will work in addition to the existing collection api rules; we also currently don't have a definition for "scopes").
